src/main/kotlin/pl/put/smartgarden/domain/user/UserService.kt

```diff
@@ -3,45 +3,29 @@ package pl.put.smartgarden.domain.user
 import org.springframework.http.HttpStatus
 import org.springframework.stereotype.Service
 import pl.put.smartgarden.domain.SmartGardenException
-import pl.put.smartgarden.domain.device.Device
-import pl.put.smartgarden.domain.device.dto.response.MeasureResponse
-import pl.put.smartgarden.domain.device.dto.response.SensorResponse
-import pl.put.smartgarden.domain.device.repository.DeviceRepository
-import pl.put.smartgarden.domain.user.dto.response.AreaResponse
-import pl.put.smartgarden.domain.user.dto.response.AreaSettingsResponse
-import pl.put.smartgarden.domain.user.dto.request.IrrigationLevelRequest
-import pl.put.smartgarden.domain.user.dto.request.LocationRequest
-import pl.put.smartgarden.domain.user.dto.request.NextIrrigationRequest
-import pl.put.smartgarden.domain.user.dto.request.UserResourceResponse
 import pl.put.smartgarden.domain.user.dto.request.UserSignInRequest
 import pl.put.smartgarden.domain.user.dto.request.UserSignInResponse
 import pl.put.smartgarden.domain.user.dto.request.UserSignUpRequest
+import pl.put.smartgarden.domain.user.dto.response.UserGeneralSettingsResponse
 import pl.put.smartgarden.domain.user.exception.UserAlreadyExistsException
 import pl.put.smartgarden.domain.user.repository.UserRepository
-import java.time.Instant
 
 @Service
 class UserService(
```

Comment:
Jestem za tym, żeby trochę jaśniej rozdzielić te serwisy - tak, żeby 1 kontroler miał swój serwis, bo zależności tych beanów serwisów mnie trochę konfundują :p

Reply:
Jak dla mnie to właśnie kontroler powinien robić za taką fasadę i nie powiedziałbym żeby było tu przesadnie dużo zależności :) A chyba lepiej żeby UserService odwoływał się do DeviceService niż DeviceRepository? Czy może lepiej stworzyć osobny serwis który będzie miał UserRepository i DeviceRepository? Ale to jak dla mnie przerost formy nad treścią - po to są te serwisy żeby ich używać IMHO :)